content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,96 @@
 ---
 id: create-a-new-react-app
-title: Create a New React App
+title: Yeni bir React Uygulaması Oluşturun
 permalink: docs/create-a-new-react-app.html
 redirect_from:
   - "docs/add-react-to-a-new-app.html"
 prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+En iyi kullanıcı ve geliştirici deneyimi için tümleşik araç kullanın.
 
-This page describes a few popular React toolchains which help with tasks like:
+Bu sayfa görevlere yardımcı olan bazı popüler araçları anlatır.
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* Birçok dosya ve bileşene ölçeklendirme.
+* Npmden üçüncü parti kütüphanelerin kullanımı.
+* Yaygın hataların erken farkedilmesi.
+* JS ve CSS'in geliştirme anında canlı olarak güncellenmesi.
+* Çıktının ürün için optimize edilmesi.
 
-The toolchains recommended on this page **don't require configuration to get started**.
+Bu sayfada tavsiye edilen araçlar **başlangıç için ayarlama gerektirmez**.
 
-## You Might Not Need a Toolchain {#you-might-not-need-a-toolchain}
+## Bir araca ihtiyacınız olmayabilir {#bir-araca-ihtiyaciniz-olmayabilir}
```

Comment:
we can capitalize first letters

Reply:
`#bir-araca-ihtiyaciniz-olmayabilir` also shouldn't be translated.

Reply:
> `#bir-araca-ihtiyaciniz-olmayabilir` also shouldn't be translated.

it's referring the section inside this page. I think it should be translated.

Reply:
We should keep the referenced naming original if you ask me.

Reply:
Oh, I didn't see this. We shouldn't translate links. Nobody did it this way. We kept those as they are in other pages. So, let's keep it same here for the sake of consistency.